packages/wdio-browserstack-service/src/service.ts

```diff
@@ -1,12 +1,72 @@
 import logger from '@wdio/logger'
-import got from 'got'
+import got, { GotRequestFunction } from 'got'
+import WebDriver from 'webdriver'
 
 import { getBrowserDescription, getBrowserCapabilities, isBrowserstackCapability } from './util'
 
 const log = logger('@wdio/browserstack-service')
 
+type Capabilities = WebDriver.Capabilities & WebdriverIO.MultiRemoteCapabilities;
+
+type Browser = WebdriverIO.BrowserObject & WebdriverIO.MultiRemoteBrowserObject;
+
+type Config = WebdriverIO.Config & { cucumberOpts?: { strict: boolean } };
+
+type Suite = {
+    title: string;
+}
+
+type Test = {
+    fullName: string,
+    parent: string,
+    title: string
+};
+
+type Context = any;
+
+type Specs = any;
+
+type Feature = {
+    document: {
+        feature: {
+            name: string;
+        }
+    }
+};
+
+type Results = {
```

Comment:
use `WebdriverIO.TestResult` instead

Reply:
Same deal, `TestResult` doesn't include the properties we use here, perhaps the one we're looking for is: https://github.com/webdriverio/webdriverio/blob/master/packages/wdio-cucumber-framework/cucumber-framework.d.ts#L15-L17

Reply:
[this](https://github.com/webdriverio/webdriverio/blob/master/packages/webdriverio/webdriverio-core.d.ts#L257-L263) interface should be used

Reply:
I think it needs some augmentation as it lacks `exception` and `status`, I'll add that!